daemon/logger/jsonfilelog/read.go

```diff
@@ -14,10 +13,11 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/daemon/logger"
+//	"github.com/docker/docker/daemon/logger/jsonfilelog/multireader"
```

Comment:
multireader is here https://github.com/moby/moby/pull/33479 - just grab the pkg and import it